packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -313,24 +324,24 @@ abstract contract MajorityVotingBase is
     }
 
     function _validateAndSetSettings(
-        uint64 _totalSupportThresholdPct,
-        uint64 _relativeSupportThresholdPct,
+        uint64 _participationThreshold,
+        uint64 _supportThreshold,
         uint64 _minDuration
     ) internal virtual {
-        if (_relativeSupportThresholdPct > PCT_BASE) {
-            revert PercentageExceeds100({limit: PCT_BASE, actual: _relativeSupportThresholdPct});
+        if (_supportThreshold > PCT_BASE) {
+            revert PercentageExceeds100({limit: PCT_BASE, actual: _supportThreshold});
         }
 
-        if (_totalSupportThresholdPct > PCT_BASE) {
-            revert PercentageExceeds100({limit: PCT_BASE, actual: _totalSupportThresholdPct});
+        if (_participationThreshold > PCT_BASE) {
+            revert PercentageExceeds100({limit: PCT_BASE, actual: _participationThreshold});
         }
 
         if (_minDuration == 0) {
             revert VoteDurationZero();
         }
```

Comment:
Related to my comment about the percentages above

Reply:
This will be addressed in https://aragonassociation.atlassian.net/browse/APP-1264.

Reply:
Not the point I'm trying to make. Same as above, we are comparing with `>` twice, whereas we should compare with `>` and `>=`